src/library.cpp

```diff
@@ -276,14 +276,33 @@ void Library::updateBookDB(const Book& book)
   Xapian::Document doc;
   indexer.set_document(doc);
 
-  const std::string title = normalizeText(book.getTitle(), lang);
-  const std::string desc = normalizeText(book.getDescription(), lang);
+  const std::string title = normalizeText(book.getTitle());
+  const std::string desc = normalizeText(book.getDescription());
+  const std::string name = normalizeText(book.getName());
+  const std::string category = normalizeText(book.getCategory());
+  const std::string publisher = normalizeText(book.getPublisher());
+  const std::string creator = normalizeText(book.getCreator());
+  const std::string tags = normalizeText(book.getTags());
   doc.add_value(0, title);
   doc.add_value(1, desc);
+  doc.add_value(2, name);
+  doc.add_value(3, category);
+  doc.add_value(4, lang);
+  doc.add_value(5, publisher);
+  doc.add_value(6, creator);
+  doc.add_value(7, tags);
```

Comment:
Do we need to store those values ?
It seems we only need the bookId stored in the data.

Reply:
Now we don't need them.

Reply:
Can you remove them so ?

Reply:
Done